web/src/javascript/constants.js

```diff
@@ -4,8 +4,8 @@ export const silverColor = "#ACACAC";
 export const silverRGBColor = "rgba(67, 67, 67, 0.1)"
 export const goldColor = "#FFD700";
 export const goldRGBColor = "rgba(100, 84, 0, 0.1)"
-export const foodMass = 9;
-export const regFoodMass = 6;
+export const resourceMass = 9;
+export const regResourceMass = 6;
```

Comment:
I know it was already like that, but I would prefer if we didn't use `reg` abbreviation